### PR TITLE
chore: github setup improvements

### PR DESCRIPTION
**Pull request by alexeagle**
See https://docs.github.com/en/github/building-a-strong-community/using-templates-to-encourage-useful-issues-and-pull-requests
This will ask users filing an issue to indicate if it's a bug or a feature request and prompt them to include answers to common questions, reducing how many round-trips they need from maintainers.
Same for pull requests, especially to think about whether this is a breaking change.

Also add a .bazelversion file to pin the version devs and CI use when developing in the repo.

In the future we might want rules_codeowners to generate the .github/CODEOWNERS file as we expand community maintainers

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+<!--
+
+If you're looking for a new rule first make sure you check awesome-bazel for an
+curated list of existing bazel rules https://github.com/jin/awesome-bazel
+
+If you don't find what you're looking for there, before opening a feature request make sure
+this repository is the right place for that.
+
+We have a list of requirements that the rules in this repository need to follow.
+You can check them in the README.md file of the project.
```
Comment (thundergolfer) [inline .github/ISSUE_TEMPLATE/feature_request.md:31]: nit: this is a bit vague about which "requirements" are being referred to in the README. Looking at the README now I can't pick out the relevant section.

Comment (alexeagle) [reply .github/ISSUE_TEMPLATE/feature_request.md:31]: oops, good catch, that's copy-pasta from rules_nodejs

**Comment by alexeagle** [comment]
@thundergolfer merge? or you can make me a contributor, either way

**Comment by thundergolfer** [comment]
both sounds good 🙂

**Pull request merged**